app/controllers/team/projects_controller.rb

```diff
@@ -2,6 +2,11 @@ class Team::ProjectsController < ApplicationController
   after_action :verify_authorized
   rescue_from ActiveRecord::RecordNotFound, with: :not_found_error
 
+  def new
+    authorize Project, :index?
+    @pager, @projects = pagy_array(ByTeamProjectFetcherService.new(current_user.team).in_progress)
```

Comment:
Should this be calling the `new` method on the Service, not `in_progress`?

Reply:
yes, this should be `new` - thanks!